tests/library/blas_dot_test.py

```diff
@@ -52,7 +52,7 @@ def make_sdfg(implementation, dtype, storage=dace.StorageType.Default):
     state.add_memlet_path(dot_node,
                           result,
                           src_conn="_result",
-                          memlet=Memlet.simple(result, "0", num_accesses=-1))
+                          memlet=Memlet.simple(result, "0", num_accesses=1))
```

Comment:
Major issue: If this goes back to -1 (memlet dynamic), code generation does not generate code (see dace/codegene/targets/cpu.py, line 837)

Reply:
it shouldn't be dynamic anyway - this is by design. The code generator does not output anything on its own if `dynamic=True`. Instead, this is done in the tasklet.